heft-plugins/heft-typescript-plugin/src/index.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+/**
+ * A Heft plugin for using TypeScript.
+ *
+ * @packageDocumentation
+ */
+
+export type {
+  IExtendedTypeScript,
+  IExtendedProgram,
+  IExtendedSourceFile
+} from './internalTypings/TypeScriptInternals';
```

Comment:
I'm a little uncomfortable with exporting these.

Reply:
Was able to move them into the lint plugins entirely